src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void write(String resultingReport, String filePath) {
+        File file = new File(filePath);
+        try {
+            Files.write(file.toPath(), resultingReport.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file, " + e);
```

Comment:
According to the checklist, exceptions in try-catch blocks should be rethrown with an informative message and the exception object. Consider including the file path in the message for more clarity. For example: throw new RuntimeException("Can't write to file at path: " + filePath, e);